lib/matplotlib/__init__.py

```diff
@@ -1476,7 +1476,6 @@ def _jupyter_nbextension_paths():
 default_test_modules = [
     'matplotlib.tests.test_png',
     'matplotlib.tests.test_units',
-    'matplotlib.tests.test_widgets',
```

Comment:
Do we *need* to keep `matplotlib.test()`?  Even if we do, can we just make `matplotlib.test()` fully rely on pytest's test discovery?